updates/32.sql

```diff
@@ -0,0 +1,12 @@
+DELETE FROM selektion_ort WHERE ID BETWEEN 39 AND 60;
```

Comment:
Note: Even though this PR is still open, i already applied the DELETE statement for selektion_ort onto the servers as discussed today.